services/commands/release_queues/parsers.go

```diff
@@ -0,0 +1,49 @@
+package release_queues
```

Comment:
So everything works even if some field are not provided, right? (since I saw some are optional)

Reply:
yep, the following release queue was created in yuri with terraform script: 96e09c49-e8b0-4352-9df7-62066b339802